### PR TITLE
Added/ changed necessary things

### PR DESCRIPTION
**Pull request by Ishika11**

### CODE REVIEW

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,27 +2,31 @@
...
   <body>
+    <style>
+      body {background-color: rgb(188, 176, 230);}
+    </style>
     <div id="content">
-      <h1>Ishika (^_^)</h1>
-      <p>Here are my favourite people:</p>
-      <p>My favourite CP programmer:</p>
-      <img src="images2/Genn.jpg"/>
+      <h1 style="color: yellow">Ishika (^_^)</h1>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html]: Remove inline css.

**Review by abhatia2107 — CHANGES_REQUESTED** [review]

**Comment by Ishika11** [comment]
Please ignore the changes in the last 2 commits, I messed a few things and was only removing the changes that happened because of it

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_index.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  font-family: 'Times New Roman', Times, serif;
+  font-size: 30px;
+  color: blue;
+}
+p.lks{
+  font-style: italic;
+}
+body {
+    background-color: rgb(177, 163, 228);
+}
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_index.css]: newline

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_my_book.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+body {
+    background-color: white;
+}
+h1{
+    color: orange;
+    font-size: 40px;
+    font-family: 'Times New Roman', Times, serif;
+}
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/style_my_book.css]: newline

**Review by abhatia2107 — APPROVED** [review]
Please add newline to end of every file before submitting. Otherwise it looks good, great work 👍

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My_book</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style_my_book.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1 style="color: Orange;">Helix - Ishika Kumar</h1>
```
Comment (abhatia2107) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/mybook.html]: Remove inline styling.

**Review by abhatia2107 — CHANGES_REQUESTED** [review]

**Review by abhatia2107 — APPROVED** [review]

**Pull request merged**